CODEOWNERS

```diff
@@ -3,4 +3,5 @@
 /pages/unitprotocol/ @milkyklim
 /pages/bancor/ @schoeler
 /pages/liquity/ @edmulraney @danielattilasimon @koltenb @RickGriff @bingen @cvalkan @bojan-liquity
-/pages/badgerdao/ @jorijnsmit
\ No newline at end of file
+/pages/badgerdao/ @jorijnsmit
+/pages/leifu/ @leifu
```

Comment:
This needs to be /pages/xdollar/

Reply:
Thanks for your remind. It has been corrected.